src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -40,6 +43,8 @@
 @Slf4j
 public class UserController {
 
+    private static final int MAX_CHILDREN_AMOUNT = 10;
```

Comment:
Why don't use it as property and inject with Value annotation?

Reply:
I will change it. But `@Value` cannot be used on static final variables and `@Size` takes just constants. Because of this I had to change constant from `@Size(min = 1, max = MAX_CHILDREN_AMOUNT...` to `@Size(min = 1, max = 10...`